doc/nrf/ug_thread_architectures.rst

```diff
@@ -113,100 +109,91 @@ For more information about the multiprotocol feature, see :ref:`ug_multiprotocol
-OpenThread can run on either the radio or the host processor.
+In co-processor designs, the application runs on one processor (the host processor) and communicates with another processor that provides the Thread radio.
+The communication happens through a serial connection using a standardized host-controller protocol (Spinel).
+
+OpenThread runs on either the radio processor or the host processor, depending on whether a network co-processor (NCP) design or a radio co-processor (RCP) design is chosen.
 
 .. _thread_architectures_designs_cp_ncp:
 
-Network Co-Processor (NCP)
+Network co-processor (NCP)
 ==========================
 
-The standard NCP design has Thread features on the SoC and runs the application layer on a host processor, which is typically more capable than the OpenThread device, although it has greater power demands.
-The host processor communicates with the OpenThread device through a serial interface (typically UART or SPI) over the Spinel protocol.
+In the standard NCP design, the network-related features run on the processor that provides the Thread radio, and the application layer runs on a host processor.
+The host processor is typically more capable than the network processor, but it has greater power demands.
+The host processor communicates with the network processor through a serial interface (typically UART or SPI) over the Spinel protocol.
 
 This design is useful for gateway devices or devices that have other processing demands, like IP cameras and speakers.
 
 This design has the following advantages:
 
-* The higher-power host can sleep, while the lower-power OpenThread device remains active to maintain its place in the Thread network.
-* Since the SoC is not tied to the application layer, development and testing of applications is independent of the OpenThread build.
-* Only the network stack and a thin application reside on the NCP, which reduces the cost of the chip (RAM and Flash usage may be smaller than in an SoC solution with the application).
-* It does not require the support for the dual-bank DFU.
-  (Host can just replace the old image with a new one.)
+* The higher-power host can sleep, while the lower-power network processor remains active to maintain its place in the Thread network.
+* Since the network processor is not tied to the application layer, development and testing of applications is independent of the OpenThread build.
+* Only the network stack and a thin application reside on the network processor, which reduces the cost of the chip.
+  RAM and flash usage are usually smaller than in a single-chip solution.
+* This design does not require support for dual-bank DFU, because the host can just replace the old image with a new one.
 
-It also has the following disadvantages:
+It has the following disadvantages:
 
-* This is the most expensive option, since it requires the application processor.
+* This is a more expensive option, because it requires a host processor for the application.
```

Comment:
Having an application on a separate processor of our choice is not only a downside, because of a higher product cost, but also a huge benefit since the application can be very complex if we decide to choose an advanced host processor.
Wondering if we could add something about it in the 'advantages' section.

Reply:
This comment is valid also for the RCP.

Reply:
Added "When choosing an advanced and powerful host processor, applications can be very complex." as advantage (for both NCP and RCP).